checkOpenPMD_h5.py

```diff
@@ -382,14 +383,16 @@ def check_root_attr(f, v, pic):
     #   optional
     result_array += test_attr(f, v, "optional", "comment", np.string_)
 
-    # Extension: ED-PIC
+    requiredExtensions = []
     if pic:
-        valid, extensionIDs = get_attr(f, "openPMDextension")
-        if valid:
-            if (ext_list[0][1] & extensionIDs) != extensionIDs:
+        requiredExtensions.append("ED-PIC")
+    valid, extensionIDs = get_attr(f, "openPMDextension")
+    if valid:
+        for extension in requiredExtensions:
```

Comment:
refactor: just check what extensions are found, enable checking for them, print which ones are found and print warnings about additional found extension that the script does not know (and will ignore)